docs/en/observability/synthetics-scaling-and-architecting.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[synthetics-scaling-and-architecting]]
+= Scaling and Architecting a Synthetics Deployment
+
+While for simple use cases little to no planning needs to be taken in the implementation of *{synthetics-app}*, 
+for complex use cases and at a high scale this section describes advanced considerations for any *{synthetics-app}*
+implementation
+
+[discrete]
+[[synthetics-no-ccs-ccr]]
+= *{synthetics-app}* does not work with CCS/CCR
```

Comment:
I don't think we need to bold the product name throughout. Also the other headings start with verbs so we could rephrase to something like this to be consistent. 

```suggestion
= Do not use the {synthetics-app} with CCS/CCR
```